src/middleware/alias.js

```diff
@@ -35,6 +35,9 @@ module.exports = function(router) {
 
   // Handle the request.
   return function aliasHandler(req, res, next) {
+    if (req.originalUrl.match(/^\/analytics/)) {
```

Comment:
This route is added through the enterprise, and I do not want to make the core be affected by the routes from there. I thought there was a reserved names area in the enterprise server that allows us to add "pre-defined" names for the forms.